specification/servicebus/resource-manager/readme.md

```diff
@@ -54,6 +54,14 @@ input-file:
 - Microsoft.ServiceBus/stable/2015-08-01/servicebus.json
 ```
 
+### Tag: package-2018-01-preview

```

Comment:
Change too the default tag at the beginning of the file.

Reply:
resolved, move to top.